src/query/src/expr.rs

```diff
@@ -11,5 +11,3 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
```

Comment:
Oops, I forget to remove this empty file...

Reply:
it was detected by git hook :joy_cat: